ocp_resources/host.py

```diff
@@ -0,0 +1,63 @@
+from ocp_resources.mtv import MTV
+from ocp_resources.resource import NamespacedResource
+
+
+class Host(NamespacedResource, MTV):
+    """
+    Migration Toolkit For Virtualization (MTV) Host resource.
+    """
+
+    api_group = NamespacedResource.ApiGroup.FORKLIFT_KONVEYOR_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        host_id=None,
+        ip_address=None,
+        provider_name=None,
+        provider_namespace=None,
+        secret_name=None,
+        secret_namespace=None,
+        client=None,
+        teardown=True,
+        yaml_file=None,
+    ):
+        super().__init__(
+            name=name,
+            namespace=namespace,
+            client=client,
+            teardown=teardown,
+            yaml_file=yaml_file,
+        )
+        self.host_id = host_id
+        self.ip_address = ip_address
+        self.provider_name = provider_name
+        self.provider_namespace = provider_namespace
+        self.secret_name = secret_name or f"{self.name}-secret"
+        self.secret_namespace = secret_namespace or self.namespace
+        self.condition_message_ready = self.ConditionMessage.HOST_READY
```

Comment:
This `self.condition_message_ready` not used and I guess it's net needed, whenever you need it you can call it via the Host resource that was created

Reply:
not sure what you mean
it is in
    def wait_for_condition_ready(self):
        self.wait_for_resource_status(
            condition_message=self.condition_message_ready,
            condition_status=self.Condition.Status.TRUE,
            condition_type=self.Condition.READY,
        )

Reply:
in mtv.py